### ISSUE TITLE
Feature: Change Entity Name in Detail-View

### ISSUE THREAD

**Issue by DaReelRigo85** [Gh61/lovelace-hue-like-light-card#184]
Hey,

first I want to say thank you for this lovely card and your work on it :)

I added it to all my Lights/Rooms and now realised that I can´t rename the entities. Now I have the problem that I don´t know what light is on. Is ist possible to add custom names to an entity like changing the Group title ?

![Screenshot 2024-02-26 082456](https://github.com/Gh61/lovelace-hue-like-light-card/assets/86531492/e204bcd2-ab64-473a-ac8d-22ef78fc07ea)

Something like this:

type: custom:hue-like-light-card
title: Wohnzimmer
icon: mdi:sofa
slider: none
scenes: []
entities:
  - entity: light.beleuchtung_wohnzimmer_deckenlampen_r
    name: Rechts
  - entity: light.beleuchtung_wohnzimmer_deckenlampen_l
    name: Links
  - entity: light.beleuchtung_wohnzimmer_deckenlampen_lv
    name: Vorne-Links
  - entity: light.beleuchtung_wohnzimmer_deckenlampen_lh
    name: Hinten-Links
  - entity: light.beleuchtung_zone_wohnzimmer_schranklampe
    name: Schrank
  - entity: light.beleuchtung_zone_wohnzimmer_fernsehbeleuchtung
    name: Fernseher

I´ve looked to the Sourcecode but because I´m new in HA I don´t know where to find this :)

thank you,
Chris

**Comment by Gh61** [comment]
You should rename your entities on Home Assistant level, so the right name is used anywhere, where you use the entity.
This can be done in Entities in the settings:
[![Open your Home Assistant instance and show your entities.](https://my.home-assistant.io/badges/entities.svg)](https://my.home-assistant.io/redirect/entities/)

Or you can simply long-press on each entity in the hue-screen. This should open `more-info` window, where you can go to the entity settings and also give the entity some nice Title.

**Comment by Sandolution** [comment]
Let me start by saying that this card rocks! I second this feature request, it would make the icing on the cake.
I ordered all of my entities using the same naming convention (type | location | name). Having 7 lights called 'ceiling' isn't very practical.

**Comment by erkr** [comment]
Yes, I also second this request. My naming is perfectly fine for most parts of HA, but some lights have names that don’t work well in the hue page for particular zones or groups.
Example: I have two aquaria, both simply named aquarium. That is a good choice per room, but not for the zone, showing aquarium twice.
To make the request more specific. Please allow to define an ‘entity object’, so optionally the name of entities can be set for a particular room:
``` 
 - entity:light.aquarium_living
    name: aquarium1
 - entity:light.aquarium_hallway
    name:  aquarium2
```

**Comment by gmaclean1981** [comment]
Was searching to see if I was missing something.  I vote for this as well!

**Comment by flosch-dev** [comment]
✋another vote for this feature, it would allow better overview w/o renaming light entities

**Comment by ziptbm** [comment]
> You should rename your entities on Home Assistant level, so the right name is used anywhere, where you use the entity. This can be done in Entities in the settings: [![Open your Home Assistant instance and show your entities.](https://camo.githubusercontent.com/101b88656d4bd28c554e41cfa1141f7d8e48b84cc5ca1ba8f80e3803f8f38a0d/68747470733a2f2f6d792e686f6d652d617373697374616e742e696f2f6261646765732f656e7469746965732e737667)](https://my.home-assistant.io/redirect/entities/)
> 
> Or you can simply long-press on each entity in the hue-screen. This should open `more-info` window, where you can go to the entity settings and also give the entity some nice Title.

Thank you for this awesome integration...I just discovered it and it's AWESOME!

As far as naming goes, I was expecting to have the ability to rename the entity like can be done with scene names.  While I respect the logic of renaming globally across your install, this might not be feasible if you have identical setups in multiple rooms. 

e.g. I have 4 bedrooms that have this identical set up with Hue lights.

```
groupEntity: light.roomA_lights
entities:
  - light.roomA_overhead_light
  - light.roomA_lamp
  - light.roomA_hue_go
```

So what this means is that I must leave the Room Name in the name otherwise, I'll have a bunch of entities all named the exact same thing.  This matters if those same entities show up in other dashboards and locations.  If I were to click into this view, I already know which room I'm in, so simply changing these only to the base indicator would be sufficient (e.g. Overhead Light, Lamp, and Hue Go).  

This same logic applies to the Scenes since each of these rooms have the same scenes, but start with the name of the room, which is why it's great this integration allows you to insert a "title: <insert name>" below each scene entity to make them easily readable.  Otherwise, you end up with a bunch of entities that all show the room name...and this is troublesome when it's a multiword room name.

Again, thanks for your consideration on this feature request and the great work on this integration.

**Comment by FreAcX** [comment]
Another vote up 👍 Would be a very nice christmas gift.

**Comment by Gh61** [comment]
This feature will be included in the next release
